_inc/client/privacy/index.jsx

```diff
@@ -52,15 +54,19 @@ class Privacy extends React.Component {
 		active: false,
 	};
 
-	togglePrivacy = () => this.props.toggleTracking( this.props.getOptionValue( 'jetpack_event_tracking' ) );
+	componentWillMount() {
+		this.props.fetchTracking();
+	}
 
 	render() {
 		const {
 			searchTerm,
 			active,
-			getOptionValue,
-			isSavingAnyOption,
 		} = this.props;
+		const isActive = ! this.props.tracking.tracks_opt_out;
+		const doToggle = () => {
```

Comment:
Just a nitpick, but is there any reason to keep this function as a constant inside the `render` method? We could have left it a method just as `togglePrivacy` was before, and just passed `this.doToggle` to `onChange`.